README.md

```diff
@@ -152,20 +181,22 @@ terminal.
 Excluding Recipe from Build
 ---------------------------
 
-Recipes can be excluded from your build by deleting their respective
-'.bb' file, or by deleting the recipe directory.
+Recipes can be excluded from your build by deleting their respective ".bb" file,
+or by deleting the recipe directory.
 
 Maintenance
 -----------
 
-Layer maintainer: Chris Conlon (<chris@wolfssl.com>)
+Layer maintainers:
+- Chris Conlon (<chris@wolfssl.com>)
+- Hayden Roche (<hayden@wolfssl.com>)
 
 https://www.wolfssl.com
 
 License
 -------
 
-wolfSSL is open source and dual licensed under both the GPLv2
-and a standard commercial license. For commercial license
-questions, please contact wolfSSL at licensing@wolfssl.com. For product
-support inquiries please contact support@wolfssl.com.
+wolfSSL is open source and dual licensed under both the GPLv2 and a standard
```

Comment:
Unrelated to your changes here, but let's add a note about the availability of our FIPS validated wolfCrypt in this license section, with a link to https://www.wolfssl.com/license/fips/.